src/core_plugins/kbn_vislib_vis_types/public/histogram.js

```diff
@@ -14,46 +14,122 @@ export default function HistogramVisType(Private) {
     'exact numbers or percentages. If you are not sure which chart you need, you could do worse than to start here.',
     params: {
       defaults: {
+        grid: {
+          categoryLines: false,
+          style: {
+            color: '#eee'
+          }
+        },
+        categoryAxes: [
+          {
+            id: 'CategoryAxis-1',
+            type: 'category',
+            position: 'bottom',
+            show: true,
+            style: {
+            },
+            scale: {
+              type: 'linear'
+            },
+            labels: {
+              show: true,
+              truncate: 100
+            },
+            title: {}
+          }
+        ],
+        valueAxes: [
+          {
+            id: 'ValueAxis-1',
+            name: 'LeftAxis-1',
+            type: 'value',
+            position: 'left',
+            show: true,
+            style: {
+            },
+            scale: {
+              type: 'linear',
+              mode: 'normal'
+            },
+            labels: {
+              show: true,
+              rotate: 0,
+              filter: false,
+              truncate: 100
+            },
+            title: {}
+          }
+        ],
+        seriesParams: [
+          {
+            show: 'true',
+            type: 'histogram',
+            mode: 'stacked',
+            data: {
+              label: 'Count'
+            },
+            drawLinesBetweenPoints: true,
+            showCircles: true
+          }
+        ],
         addTooltip: true,
         addLegend: true,
         legendPosition: 'right',
+        showCircles: true,
+        interpolate: 'linear',
         scale: 'linear',
-        mode: 'stacked',
+        drawLinesBetweenPoints: true,
+        radiusRatio: 9,
         times: [],
         addTimeMarker: false,
         defaultYExtents: false,
         setYExtents: false
       },
-      legendPositions: [{
-        value: 'left',
-        text: 'left',
+      positions: ['top', 'left', 'right', 'bottom'],
+      chartTypes: ['line', 'area', 'histogram'],
+      axisModes: ['normal', 'percentage', 'wiggle', 'silhouette'],
+      scaleTypes: ['linear', 'log', 'square root'],
+      chartModes: ['normal', 'stacked'],
+      interpolationModes: [{
+        value: 'linear',
+        text: 'straight',
       }, {
-        value: 'right',
-        text: 'right',
+        value: 'cardinal',
+        text: 'smoothed',
       }, {
-        value: 'top',
-        text: 'top',
-      }, {
-        value: 'bottom',
-        text: 'bottom',
+        value: 'step-after',
+        text: 'stepped',
       }],
-      scales: ['linear', 'log', 'square root'],
-      modes: ['stacked', 'percentage', 'grouped'],
-      editor: histogramTemplate
+      editor: pointSeriesTemplate,
+      optionTabs: [
+        {
+          name: 'advanced',
+          title: 'Axes & Metrics',
```

Comment:
consider Metrics & Axis, given the first section is metrics, and the second is axes